spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/AuditRecord.java

```diff
@@ -58,7 +59,7 @@ public class AuditRecord {
 	private String correlationId;
 
 	@Lob
-	@Type(type = "org.hibernate.type.TextType")
+	@JdbcTypeCode(Types.LONGVARCHAR)
```

Comment:
I know we had a headache around LOB support specifically w/ Postgres. Hopefully the DB ITs will and ITs will ferret this out if it is a problem. The pain w/ the LOB support was that not only was it different across vendors, but also versions of a vendor.

Reply:
Running Postgres Smoke Tests locally will quickly identify if this is a valid change.